src/umbraco.businesslogic/UserGroup.cs

```diff
@@ -0,0 +1,309 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Umbraco.Core;
+using Umbraco.Core.Events;
+
+namespace umbraco.BusinessLogic
+{
+    /// <summary>
+    /// Represents a umbraco Usergroup
+    /// </summary>
+    [Obsolete("Use the UserService instead")]
```

Comment:
Been a while now... but think I was probably just following existing patterns.  There was likely the analogous class for User or UserType, and if in use - even if obsolete - probably thought needed to be there (but with the same warning that it'll likely be going soon and there are better public methods to use).  If you are overhauling this now anyway sure there's no reason now to keep it.